projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -3574,7 +3574,7 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
      * Sets this._height
      */
     protected _derivePossibleHeight() {
-        if ((this._height && this._height.indexOf('%') === -1) || !this._height || !this.isAttachedToDom) {
+        if ((this._height && this._height.indexOf('%') === -1) || !this._height || !this.isAttachedToDom || this.rowBasedHeight === 0) {
```

Comment:
Please transfer your changes from the 6.2.x branch here.

Reply:
@kdinev Done